great_expectations/datasource/data_connector/runtime_data_connector.py

```diff
@@ -51,9 +54,80 @@ def __init__(
             batch_spec_passthrough=batch_spec_passthrough,
         )
 
-        self._batch_identifiers = batch_identifiers
+        if assets is None:
+            assets = {}
+        self._assets: Dict[str, Union[dict, Asset]] = assets
+        self._batch_identifiers: dict = {}
+
+        self._build_assets_from_config(config=assets)
+
+        # add batch_identifiers defined at the DataConnector level.
+        self._add_batch_identifiers(batch_identifiers)
```

Comment:
@Shinnnyshinshin Any chance we can use "keyword" style arguments (for somewhat better readability)?  Thanks!

Reply:
@Shinnnyshinshin Whenever there is time, I was curious to understand why we want to get rid of `batch_identifiers` at the `DataConnector` level.  Wouldn't we want to be able to have these general `batch_identifiers` apply to all assets, if appropriate -- and thus reduce the need to configure the same `batch_identifiers` for each `Asset`?  Thanks!

Reply:
Good point @alexsherstinsky, I've made the change according to your first point, and would be happy to discuss your second question.